tests/fixtures/fixtures.py

```diff
@@ -145,18 +145,18 @@ def _session_tiny_bert_config():  # type: ignore
 
 
 @pytest.fixture(scope='session')
-def _session_gpt2_config():  # type: ignore
+def _session_tiny_gpt2_config():  # type: ignore
     transformers = pytest.importorskip('transformers')
 
-    hf_config = transformers.AutoConfig.from_pretrained('gpt2')
+    hf_config = transformers.AutoConfig.from_pretrained('sshleifer/tiny-gpt2')
```

Comment:
Is this model 'sshleifer/tiny-gpt2' going to exist in future releases of transformers? Can new releases silently remove this? This doesn't seem like one of the standard models so may get removed. Other than this issue, it looks fine.

Reply:
ah good point that it could get removed. let me switch to the standard ones and just modify the params to make tiny models.